core-bundle/src/Resources/contao/templates/backend/be_confirm.html5

```diff
@@ -37,7 +37,7 @@
       <form class="tl_form" method="post">
         <div class="tl_formbody_edit">
           <input type="hidden" name="FORM_SUBMIT" value="invalid_token_url">
-          <input type="hidden" name="REQUEST_TOKEN" value="<?= REQUEST_TOKEN ?>">
+          <input type="hidden" name="REQUEST_TOKEN" value="<?= Contao\System::getContainer()->get('contao.csrf.token_manager')->getDefaultTokenValue() ?>">
```

Comment:
I wonder if we should have a helper in `Template` for that. Or use an insert tag or so for that. Just think about all the users that need to change this in their templates. Looks weird to access the container. In Twig it would be/is gonna be a global function anyway, I guess? /cc @m-vo

Reply:
👍  for helper method in `Template`. Regarding the insert tag: we already had `{{request_token}}` but we deprecated it in Contao 4.13 and removed it in Contao 5.0.

Reply:
Back then we argued, that the value should always be provided by the controller though, not by the template itself: https://github.com/contao/contao/pull/3620

Reply:
Yes, it should be provided by the controller. Should I do that in this PR, or in a seperate one?

Reply:
I‘d say in this one.

Reply:
Done 37cc24572acc832749b3aba57a6ae1393cf7272b